README.md

```diff
@@ -63,6 +63,18 @@ plugins = strawberry.ext.mypy_plugin
 
 [mypy]: http://www.mypy-lang.org/
 
+### Django Integration
```

Comment:
cool! can you also add an introduction specifying that this for now includes "only" a django view for serving the api?

I also think we need to add the app to the installed apps so the template can be found :)